pkgs/applications/graphics/inkscape/default.nix

```
@@ -58,11 +59,11 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "inkscape";
-  version = "1.2.2";
+  version = "1.3";
 
   src = fetchurl {
     url = "https://media.inkscape.org/dl/resources/file/inkscape-${version}.tar.xz";
```

The release notes at https://gitlab.com/inkscape/inkscape/-/releases/INKSCAPE_1_3 suggest to switch to this scheme instead:

```suggestion
    url = "https://inkscape.org/release/inkscape-${version}/source/archive/xz/dl/inkscape-${version}.tar.xz";
```

---

I will do the fix when the other two issues upstream get fixed.